### PR TITLE
Fix MySQL command to set password for user, the existing command is not working on MySQL v8 

### PR DESCRIPTION
**Pull request by deveshub**
Dear Vlad,

I'm taking your High-Performance Java Persistence course. While setting up the workspace with MySQL, I encountered an error in one of the MySQL commands. Raising this Pull Request to get it corrected. 

Regards,
Devesh Mankar

### CODE REVIEW

**Comment by vladmihalcea** [comment]
Thanks for taking my course. 

I don't understand why this change is necessary, so would you mind explaining why a hash would solve thr issue you got?

Also, would you mind elaborating the issue as everything works fine for me?

**Comment by deveshub** [comment]
Issue: 
In the original README.md,  at line #131, there's the following command: 

`SET PASSWORD for 'mysql'@'localhost' = PASSWORD('admin');`

after executing this command MySQL produces the following error on version 8 : 

`Error Code: 1064. You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'PASSWORD('admin')' at line 1`

To fix this, the syntax needs to be corrected to the following: 

`SET PASSWORD for 'mysql'@'localhost'='admin';`

In short, need to remove the PASSWORD() clause from the statement. 

If it is working for you, could you please share your MySQL version, so I can modify this and add a comment mentioning for MySQL 8 to execute the correct command?

**Comment by vladmihalcea** [comment]
Thanks for the clarification. It was indeed documented some time ago fof MySQL 5.7. I'll change it after I test it too.

**Comment by deveshub** [comment]
Thanks for taking a note. 
Since you're planning to change it, in the same section, additionally, on line # 120 you've recommended installing MySQL 8, hence you need not mention the version for below command and the hyperlink on line #140 is broken, you may want to check if the correct link can be updated there.

**Comment by getamihalcea** [comment]
Thanks for the hint. I'll fix that as well.

**Comment by vladmihalcea** [comment]
Thanks, I merged it.
